app/routes.ts

```diff
@@ -1,14 +1,14 @@
 import { Application } from 'express';
-
 import { healthCheck } from './controllers/healthCheck';
 import { getUsers, getUserById, createUser } from './controllers/users';
 import { getTodos } from './controllers/todos';
 import { getCardsInfo, getCards } from './controllers/cards';
+import { userHandlMiddleware } from './middlewares/user';
 
 export const init = (app: Application): void => {
   app.get('/health', healthCheck);
   app.get('/users', getUsers);
-  app.post('/users', createUser);
+  app.post('/users', userHandlMiddleware, createUser);
```

Comment:
please use descriptive names, for example validateUser

Reply:
changed name